04-files.md

```diff
@@ -16,8 +16,8 @@ The only thing that's missing is a library with a rather unpleasant name:
 import glob
 ~~~
 
-The `glob` library contains a single function, also called `glob`,
```

Comment:
Do you disagree with single or think it is redundant? I like the emphasis is places.